### ISSUE TITLE
Word of the day

### ISSUE THREAD

**Issue by marcinpawlukiewicz** [Hugo0/wordle#20]
Reiss is not a Polish word. It's a British clothing brand. We don't even have that clothing brand in Poland.

**Comment by kiyas** [comment]
Yes, words are autoparsed from blogs, wikis etc. and there are lots of errors especially in agglutinative languages. For example, Turkmen words dictionary here are formed by dropping suffixes, so after failing to find the word, you realize that it's a typo.

**Comment by Hugo0** [comment]
Would be awesome if you guys provide a better wordlist!

**Comment by Hugo0** [comment]
Thanks for reporting! The Polish word list has been significantly improved with native frequency data and a dedicated blocklist (PR #118, #115). Proper nouns and brand names should no longer appear as daily words.
